components/calendar/src/iso.rs

```diff
@@ -266,8 +266,13 @@ impl Iso {
 
     // Fixed is day count representation of calendars starting from Jan 1st of year 1.
     // The fixed calculations algorithms are from the Calendrical Calculations book.
-    pub(crate) fn fixed_from_iso(date: IsoDateInner) -> RataDie {
-        calendrical_calculations::iso::fixed_from_iso(date.0.year, date.0.month, date.0.day)
+    #[doc(hidden)]
+    pub fn to_fixed(date: Date<Iso>) -> RataDie {
+        calendrical_calculations::iso::fixed_from_iso(
+            date.inner.0.year,
+            date.inner.0.month,
+            date.inner.0.day,
+        )
```

Comment:
Nit: I think `fixed_from_iso` is a string from Reingold.

I suggest adding new public functions to `Date<Iso>` called `to_rata_die` and `from_rata_die`.

Reply:
and the method in `calendrical_calculations` will keep that name

Reply:
let's not nitpick non-public names on this PR please, if you want to stabilise them please file an issue

Reply:
ok, you know how I feel about `#[doc(hidden)]`. Can you at least reduce the diff and use the existing names please?

Reply:
this goes from one doc-hidden API to two, because I need the other direction.

the diff is how it is because this crate previously had two `to_fixed` methods, now it has one

Reply:
I don't have time right now to review the big diff. I'll get to it soon, or sooner if the PR gets a smaller diff.

Reply:
it's some tests that compile and pass